### ISSUE TITLE
grype fixed version cyclonedxjson

### ISSUE THREAD

**Issue by cpendery** [anchore/grype#762]
**What would you like to be added**:

Add a fixed version for grype. It makes sense to me to add in under the properties of a vulnerability. 

```json
{
  "name": "grype:fixed_versions",
  "value": "v0.1.2,v01.3"
}
```

**Why is this needed**:

It is given in the `-o json` output and not in the cyclonedx json output

**Additional context**:
I have an issue here https://github.com/CycloneDX/cyclonedx-property-taxonomy/issues/12 to add a grype namespace. Please feel free to comment on it for context or if you want to reuse the syft namespace

**Comment by nicoskiy** [comment]
^ I do not see this enhancement in v0.104.1 version of grype on the --output cyclonedx-json format. 
I do not see the properties field at all in the vulnerabilities section of the cyclonedx json output

**Comment by nicoskiy** [comment]
Example command
`grype --output cyclonedx-json python@sha256:01f36278bcca9fb4a5bdda7edc85be44218190a9371f642d8f852f36ad9fa23d`

Example output in vulnerabilities section:

```
{
      "bom-ref": "urn:uuid:37ab927c-c344-434b-98d8-870ca1b1c763",
      "id": "CVE-2025-64720",
      "source": {
        "name": "debian-distro-debian-13",
        "url": "https://security-tracker.debian.org/tracker/CVE-2025-64720"
      },
      "references": [
        {
          "id": "CVE-2025-64720",
          "source": {
            "name": "debian-distro-debian-13",
            "url": "https://security-tracker.debian.org/tracker/CVE-2025-64720"
          }
        }
      ],
      "ratings": [
        {
          "score": 7.1,
          "severity": "high",
          "method": "CVSSv31",
          "vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:U/C:L/I:N/A:H"
        },
        {
          "source": {
            "name": "FIRST",
            "url": "https://www.first.org/epss/"
          },
          "score": 0.00033,
          "method": "other"
        }
      ],
      "description": "LIBPNG is a reference library for use in applications that read, create, and manipulate PNG (Portable Network Graphics) raster image files. From version 1.6.0 to before 1.6.51, an out-of-bounds read vulnerability exists in png_image_read_composite when processing palette images with PNG_FLAG_OPTIMIZE_ALPHA enabled. The palette compositing code in png_init_read_transformations incorrectly applies background compositing during premultiplication, violating the invariant component ≤ alpha × 257 required by the simplified PNG API. This issue has been patched in version 1.6.51.",
      "affects": [
        {
          "ref": "pkg:deb/debian/libpng16-16t64@1.6.48-1?arch=amd64&distro=debian-13&package-id=790f76ab02b087c3&upstream=libpng1.6"
        }
      ]
}
```

This vulnerability does have a fix (https://security-tracker.debian.org/tracker/CVE-2025-64720) but no properties and no fix version in the grype output

**Comment by nicoskiy** [comment]
Ahh I do see this
https://github.com/anchore/grype/blob/main/grype/presenter/cyclonedx/vulnerability.go#L72

Has this enhancement been removed?
